core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/Tenant.java

```diff
@@ -32,8 +35,10 @@
 
     private String id;
 
+    @JsonInclude(Include.NON_EMPTY)
     private List<AggregationTemplate> aggregationTemplates = new ArrayList<>();
```

Comment:
I have expressed similar concern about introducing Jackson dependency in the core library. It would be worthwhile to see if we can do things in a way to avoid that.